src/Raven.Server/Monitoring/Snmp/SnmpOids.cs

```diff
@@ -661,6 +859,16 @@ public static DynamicJsonArray ToJson()
 
                     return array;
                 }
+
+                public static async Task WriteMibAsync(SnmpMibWriter writer)
+                {
+                    foreach (var field in typeof(General).GetFields())
+                    {
+                        var fieldValue = GetFieldValue(field);
+
+                        await writer.WriteObjectAsync(field.Name, "server", fieldValue.TypeCode.Value, fieldValue.Description, fieldValue.Oid);
```

Comment:
```suggestion
                        await writer.WriteObjectAsync(field.Name, "databases", fieldValue.TypeCode.Value, fieldValue.Description, fieldValue.Oid);
```

?

Reply:
it is actually correct, because the OID suffix in SnmpOids for this category already contains that '5' for databases (e.g. `public const string FaultedCount = "5.1.10";`) so I'm pointing to parent, which is 'server'

Reply:
ok just wanted to make sure